website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -648,7 +724,7 @@ run `terraform destroy`.
 To configure a collection to enable ORCA, add the line
 `"granuleRecoveryWorkflow": "DrRecoveryWorkflow"` to the collection configuration
 as seen below. Optionally, you can exclude files by adding values to an
-`"exclue_file_type"` variable. For more information, see the documentation on the
+`"excludeFfileType"` variable. For more information, see the documentation on the
```

Comment:
typo `excludeFfileType` should be `excludeFileTypes`

Reply:
Edited typo to excludeFileTypes.